.changeset/shaggy-actors-call.md

```diff
@@ -0,0 +1,5 @@
+---
+"@layerzerolabs/oft-evm": minor
```

Comment:
@TRileySchwarz @ryandgoulding @janjakubnanista Just want to confirm that this change would be considered minor?

Reply:
Confirmed with @janjakubnanista that yes it would be a minor change